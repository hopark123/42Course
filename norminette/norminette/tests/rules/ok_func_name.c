int	func(void);

int	func2(void)
{
	return (1);
}
