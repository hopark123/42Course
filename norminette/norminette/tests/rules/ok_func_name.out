[36mok_func_name.c[0m - [32mIsFuncPrototype[0m In "GlobalScope" from "None" line 1":
		<INT> <TAB> <IDENTIFIER=func> <LPARENTHESIS> <VOID> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
[36mok_func_name.c[0m - [32mIsEmptyLine[0m In "GlobalScope" from "None" line 2":
		<NEWLINE>
[36mok_func_name.c[0m - [32mIsFuncDeclaration[0m In "GlobalScope" from "None" line 3":
		<INT> <TAB> <IDENTIFIER=func2> <LPARENTHESIS> <VOID> <RPARENTHESIS> <NEWLINE>
[36mok_func_name.c[0m - [32mIsBlockStart[0m In "Function" from "GlobalScope" line 4":
		<LBRACE> <NEWLINE>
[36mok_func_name.c[0m - [32mIsExpressionStatement[0m In "Function" from "GlobalScope" line 5":
		<TAB> <RETURN> <SPACE> <LPARENTHESIS> <CONSTANT=1> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
[36mok_func_name.c[0m - [32mIsBlockEnd[0m In "Function" from "GlobalScope" line 6":
		<RBRACE> <NEWLINE>
ok_func_name.c: OK!
